content/en/docs/concepts/scheduling/scheduling-framework.md

```diff
@@ -140,53 +139,53 @@ to prevent race conditions while the scheduler waits for the bind to succeed.
 
 This is the last step in a scheduling cycle. Once a Pod is in the reserved
 state, it will either trigger [Unreserve](#unreserve) plugins (on failure) or
-[Post-bind](#post-bind) plugins (on success) at the end of the binding cycle.
-
-*Note: This concept used to be referred to as "assume".*
+[PostBind](#post-bind) plugins (on success) at the end of the binding cycle.
 
 ### Permit
 
-These plugins are used to prevent or delay the binding of a Pod. A permit plugin
-can do one of three things.
+_Permit_ plugins are invoked at the end of the scheduling cycle for each Pod, to
+prevent or delay the binding to the candidate node. A permit plugin can do one of
+the three things:
 
 1.  **approve** \
-    Once all permit plugins approve a Pod, it is sent for binding.
+    Once all Permit plugins approve a Pod, it is sent for binding.
 
 1.  **deny** \
-    If any permit plugin denies a Pod, it is returned to the scheduling queue.
+    If any Permit plugin denies a Pod, it is returned to the scheduling queue.
     This will trigger [Unreserve](#unreserve) plugins.
 
 1.  **wait** (with a timeout) \
-    If a permit plugin returns "wait", then the Pod is kept in the permit phase
-    until a [plugin approves it](#frameworkhandle). If a timeout occurs, **wait**
-    becomes **deny** and the Pod is returned to the scheduling queue, triggering
-    [Unreserve](#unreserve) plugins.
+    If a Permit plugin returns "wait", then the Pod is kept in an internal "waiting"
+    Pods list, and the binding cycle of this Pod starts but directly blocks until it
+    gets [approved](#frameworkhandle). If a timeout occurs, **wait** becomes **deny**
+    and the Pod is returned to the scheduling queue, triggering [Unreserve](#unreserve)
+    plugins.
 
-**Approving a Pod binding**
+{{< note >}}
+While any plugin can access the list of "waiting" Pods and approve them
+(see [`FrameworkHandle`](#frameworkhandle)), we expect only the permit
+plugins to approve binding of reserved Pods that are in "waiting" state. Once a Pod
+is approved, it is sent to the [PreBind](#pre-bind) phase.
```

Comment:
It's slightly better to [address the reader as “you”](https://kubernetes.io/docs/contribute/style/style-guide/#address-the-reader-as-you) _but_ let's not have that detail block a merge.